infrastructure/resource_group_level/modules/webapps/trade-finance-manager-api.bicep

```diff
@@ -50,6 +48,10 @@ param secureConnectionStrings object = {
 @secure()
 param additionalSecureConnectionStrings object = {
   GOV_NOTIFY_EMAIL_RECIPIENT: 'test-value'
+  // TODO:FN-429 Note that this has a value like https://tfs-dev-tfm-fd.azurefd.net 
+  // while in the CLI it is injected as a secret, we can probably calculate it from the Front Door component.
+  // Note that it has been replaced by TFM_UI_URL in the CLI scripts
+  TFM_URI: 'test-value'
```

Comment:
I suspect this value isn't needed and can go.

Reply:
yep, we can remove it in this PR

Reply:
Ok - I will remove it. I think it was changed to `TFM_UI_URL` not the API one, right?

Reply:
ah yes. you are on TFM API bicep file already. so `TFM_UI_URL` it is

Reply:
ok! Note that the change has not been made in the configuration - I can still see `TFM_URI` in the connection strings area and not `TFM_UI_URL` (or even `TFM_API_URL`).